vllm/worker/model_runner.py

```diff
@@ -82,11 +82,18 @@ def load_model(self) -> None:
         vocab_size = self.model.config.vocab_size
 
         if self.lora_config:
+            assert hasattr(
+                self.model, "supported_lora_modules"
+            ) and self.model.supported_lora_modules, "Model does not support LoRA"
+            assert hasattr(self.model, "embedding_modules") and hasattr(
```

Comment:
Nit: Can you split this into two asserts? That will make the error message a little clearer if one of them is missing (for debugging)